framework/src/main/java/org/checkerframework/framework/type/DefaultAnnotatedTypeFormatter.java

```diff
@@ -270,10 +270,16 @@ public String visitExecutable(AnnotatedExecutableType type, Set<AnnotatedTypeMir
         sb.append("METHOD");
       }
       sb.append('(');
-      AnnotatedDeclaredType rcv = type.getReceiverType();
-      if (rcv != null) {
-        sb.append(visit(rcv, visiting));
-        sb.append(" this");
+      AnnotatedDeclaredType rcv;
+      try {
+        rcv = type.getReceiverType();
```

Comment:
Is the NPE in `type.getReceiverType()` or the call to `visit` below?

Reply:
A stack trace indicates it is in `getReceiverType()`.  This only happens under JDK 18.